runtime/moonbase/src/lib.rs

```diff
@@ -1398,6 +1398,7 @@ impl Contains<Call> for MaintenanceFilter {
 			Call::Ethereum(_) => false,
 			Call::EVM(_) => false,
 			Call::XTokens(_) => false,
+			Call::XcmTransactor(_) => false,
```

Comment:
Good catch. I wonder if it would  make more sense to explicitly allow things we want to still be enabled, and everything else defaults to false.

Reply:
I think for maintenance it is better yes, but we have to be sure of the correct list.
Do we have tests for maintenance ? (cc @joelamouche )

Reply:
At least Assets should also be in this list, so I will add it.